fix: update disability max weekly benefit

heykarimoff opened the pull request:
* Minor change.
* Impacted periods: all.
* Impacted areas: `openfisca_aotearoa/parameters/disability_allowance/max_weekly_benefit.yaml`
* Details:
  - Disability Allowance Max Weekly Benefit rate was outdated, updated it.

- - - -

These changes _(remove lines which are not relevant to your contribution)_:

- Fix or improve an already existing calculation.

<img width="980" alt="Google Chrome 2024-09-05 12 45 22" src="https://github.com/user-attachments/assets/8170ee69-83e8-423f-bf0c-e83e75561cf0">

heykarimoff commented:
@verbman I updated the existing rate. I'm not sure if that's right way. I thought about adding a new period with value as well.

verbman reviewed (commented):
There's a few things going on here, firstly the rate is wrongly applied as it does need to be date based like the others, the file itself isn't in the correct location (along with some others that predate the structure as it's been adopted). I'd be happy to tidy this up over the next few days as a new PR as it's somewhat convoluted.

verbman commented:
If you're wanting to take it further I'd recommend checking the https://github.com/digitalaotearoa/openfisca-aotearoa/blob/main/openfisca_aotearoa/parameters/social_security/child_disability_allowance/base.yaml file as a starter to see where to get the dates and amounts. Note that this does require more than just a copy and replace as sometimes rates aren't all changed at the same time.
If also should be in the `parameters/social_security/disability_allowance` folder. I note that that folder already exists and it's contents related to child_disability_allowance meaning it is also in the wrong location and needs moving as well.
This same issue also relates to the contents of `parameters/entitlements` which exists from the very early days of the project before the current structure was adopted. I've intended to tidy up all this at some point - it perhaps needs to be reported as an issue on the repository.

heykarimoff commented:
> There's a few things going on here, firstly the rate is wrongly applied as it does need to be date based like the others, the file itself isn't in the correct location (along with some others that predate the structure as it's been adopted). I'd be happy to tidy this up over the next few days as a new PR as it's somewhat convoluted.

@verbman It would be great if you tide this up. I will wait your PR.

> If you're wanting to take it further I'd recommend checking the https://github.com/digitalaotearoa/openfisca-aotearoa/blob/main/openfisca_aotearoa/parameters/social_security/child_disability_allowance/base.yaml file as a starter to see where to get the dates and amounts. Note that this does require more than just a copy and replace as sometimes rates aren't all changed at the same time. If also should be in the `parameters/social_security/disability_allowance` folder. I note that that folder already exists and it's contents related to child_disability_allowance meaning it is also in the wrong location and needs moving as well. This same issue also relates to the contents of `parameters/entitlements` which exists from the very early days of the project before the current structure was adopted. I've intended to tidy up all this at some point - it perhaps needs to be reported as an issue on the repository.

I think that [social_security/disability_allowance/base.yaml](https://github.com/digitalaotearoa/openfisca-aotearoa/blob/main/openfisca_aotearoa/parameters/social_security/disability_allowance/base.yaml) and [social_security/child_disability_allowance/base.yaml](https://github.com/digitalaotearoa/openfisca-aotearoa/blob/main/openfisca_aotearoa/parameters/social_security/child_disability_allowance/base.yaml)  are part of the same legislation. Their content is correct.

I updated the disability allowance benefit formula in [social_security/disability_allowance/disability_allowance.py](https://github.com/digitalaotearoa/openfisca-aotearoa/blob/main/openfisca_aotearoa/variables/acts/social_security/disability_allowance/disability_allowance.py) and deleted [disability_allowance/max_weekly_benefit.yaml](https://github.com/digitalaotearoa/openfisca-aotearoa/blob/main/openfisca_aotearoa/parameters/disability_allowance/max_weekly_benefit.yaml). Let me know what you think. If this is wrong then I will wait you PR.

verbman reviewed (approved):
Nice work, good to get this inconsistency tidied up
